README.md

```diff
@@ -472,6 +472,7 @@ Int. J. of Computer Vision (IJCV), 2018.
 - <a name="Shedligeri18arxiv"></a>Shedligeri, P.A., Shah, K., Kumar, D., Mitra, K.,  
 *[Photorealistic Image Reconstruction from Hybrid Intensity and Event based Sensor](https://arxiv.org/pdf/1805.06140.pdf)*,  
 arXiv:1805.06140, 2018.
+- <a name="Scheerlinck18accv"></a>Scheerlinck, C., Barnes, N., Mahony, R., *[Continuous-time Intensity Estimation Using Event Cameras](https://cedric-scheerlinck.github.io/files/2018_scheerlinck_continuous-time_intensity_estimation.pdf)*, Asian Conf. Computer Vision (ACCV), 2018. ([Webpage](https://cedric-scheerlinck.github.io/continuous-time-intensity-estimation))
```

Comment:
Cedric, we like the authors, title and publication venue in separate lines..
You need to add 2 blank spaces at the end of each line to force a line break. thanks. You can see the preview with the "View" button.